README.md

```diff
@@ -35,10 +35,18 @@ kubectl create secret generic ml-admin-secrets \
 Refer to the official Kubernetes documentation for detailed steps on how to [create a secret](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/#create-a-secret).
 
 4. Create a `values.yaml` file to customize the settings. Specify the number of pods (one MarkLogic host in this case), add the secret name for the admin credentials (if not using the automatically generated one), and specify the resources that should be allocated to each MarkLiogic pod.
+
+Note: Please make sure to use latest MarkLogic Docker image for the new implementation as shown below in the values.yaml file:
 ```
 # Create a single MarkLogic pod
 replicaCount: 1
 
+# Marklogic image parameters
+image:
+  repository: marklogicdb/marklogic-db;
+  tag: latest
```

Comment:
We recommend to not use the tag latest.
We should put the latest versioned tag instead.

Reply:
updated tag